README.md

```diff
@@ -7,14 +7,62 @@ each change. These instructions assume you are on a Mac and that you have Docker
 working. While docker-machine is generally preferable if you have significant workloads,
 Docker Desktop will be sufficient for frontend stuff.
 
-Steps to configure:
+### Preface
+Before starting, ensure that you have set up your PIC-SURE All-In-One virtual box environment with a bridge network, 
+that you can navigate to your PIC-SURE environment using the IP Address on your virtual box and have logged in. 
+Additionally, note that this guide was created for the macOS.
 
-./test_using_remote_backend.sh
+### Guide
+1. Visit your virtualbox PIC-SURE All-In-One page, inspect, and open the network tab.
+    1. Reload the page to allow the network tab to populate.
+    2. Search for and download `settings.json`.
+2. Open the browser certificates for the current page and download `picsure.local.cert`.
+    1. Add `picsure.local.cert` to your local security chain.
+        1. Double-click on the `picsure.local` in your security chain.
+        2. Expand the `Trust` section.
+        3. Select  `When using this certificate: Always Trust`.
```

Comment:
The frontend harness isn't really related to the all in one. The frontend harness's  role is really for developing using a remote backend, for example some ec2 running one of our httpd servers.This section should be removed. 
I just had you set yours up to point to your local all in one because then you control everything.

Reply:
We can make a guide for internal use to do these steps

Reply:
I will separate it out into a google doc for internal use.